cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -114,6 +114,7 @@ def handle_grading_policy_changed(sender, **kwargs):
     """
     kwargs = {
         'course_key': unicode(kwargs.get('course_key')),
+        'grading_policy_hash': unicode(kwargs.get('grading_policy_hash')),
```

Comment:
@awaisdar001 I am using `grading_policy_hash` here in the receiver.

Reply:
but it is passed again in kwargs in `compute_all_grades_for_course` right?
But in `compute_all_grades_for_course` we are not making use of this kwarg

Reply:
kwargs are used in the below log which is:
```
log.info("Grades: Created {task_name}[{task_id}] with arguments {kwargs}".format(
         task_name=compute_all_grades_for_course.name,
         task_id=result.task_id,
         kwargs=kwargs,
     ))
```

Reply:
best got it.